packages/plugin/webpack/src/inject/protocol-loader.ts

```diff
@@ -0,0 +1,63 @@
+import * as path from 'path';
+
+// eslint-disable-next-line node/no-unpublished-import
+import { protocol } from 'electron';
+
+import type { WebpackPluginConfig } from '../Config';
+
+type InternalConfig = Required<Required<WebpackPluginConfig>['customProtocolForPackagedAssets']>;
+declare const __ELECTRON_FORGE_INTERNAL_PROTOCOL_CONFIGURATION__: InternalConfig;
+
+const config: InternalConfig = __ELECTRON_FORGE_INTERNAL_PROTOCOL_CONFIGURATION__ as any;
+
+const appRoot = path.join(__dirname, '..');
+const rendererRoot = path.join(appRoot, 'renderer');
+
+const STATUS_CODE_BAD_REQUEST = 400;
+const STATUS_CODE_FORBIDDEN = 403;
+const STATUS_CODE_INTERNAL_SERVER_ERROR = 500;
+
+protocol.registerFileProtocol(config.protocolName, (request, cb) => {
```

Comment:
Isn't this call deprecated in the recent Electron versions and instead one should use "handle"?

Reply:
Yeah, but it's not gonna be removed any time soon and we can migrate at some point in the future.  This at the moment is more performant that `handle` for complicated reasons so to avoid issues around that we'll use the deprecated API for a bit